### PR TITLE
JsonPath like in payload selector

### PR DESCRIPTION
**Pull request by agourlay**

### CODE REVIEW

File: qdrant-landing/content/documentation/concepts/search.md [hunk 1]
```diff
@@ -237,6 +237,13 @@ client.search(
 )
 ```
 
+It is possible to target nested fields using a dot notation:
+- `payload.nested_field` - for a nested field
+- `payload.nested_array[].sub_field` - for projecting nested fields within an array
```
Comment (timvisee) [inline qdrant-landing/content/documentation/concepts/search.md:242]: Should this also mention `include`?

File: qdrant-landing/content/documentation/concepts/search.md [hunk 2]
```diff
@@ -237,6 +237,13 @@ client.search(
...
+
+Accessing array elements by index is currently not supported.
+
+
```
Comment (timvisee) [inline qdrant-landing/content/documentation/concepts/search.md:246]:
```suggestion
```

File: qdrant-landing/content/documentation/concepts/search.md (hunk 1 above)
Comment (agourlay) [reply qdrant-landing/content/documentation/concepts/search.md:242]: There is a doc about include vs exclude a few lines up.
Is it what you mean?

Comment (timvisee) [reply qdrant-landing/content/documentation/concepts/search.md:242]: You're right! I quickly skimmed the code, but missed it.

I was also under the assumption that you could use include and exclude together, but it appears I'm wrong on that.

I added a commit to your branch to format `include` and `exclude` as literals. I think that clarifies it a bit. Hope you don't mind :smiley:

**Review by timvisee — APPROVED** [review]

**Pull request merged**